.github/workflows/conda.yml

```
@@ -0,0 +1,59 @@
+name: Build conda nightlies
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# When this workflow is queued, automatically cancel any previous running
+# or pending jobs from the same branch
+concurrency:
+  group: conda-${{ github.head_ref }}
+  cancel-in-progress: true
+
+# Required shell entrypoint to have properly activated conda environments
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  conda:
+    name: Build (and optionally upload) conda nightlies
```

The name that appears on PR checks, `Build conda nightlies / Build (and optionally upload) conda nightlies`, is fairly long. Maybe we could shorten the job name and still convey most of the overall message? 

```suggestion
    name: build
```

---

Maybe "Build (and upload)"?